run_full_pipe.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# this is a gold standard reproducible pipeline for the analysis of Live-Cell timelapse imaging data
+
+
+############################################
+# Download and pre-process the data
+############################################
+echo "Starting the data download and pre-processing..."
+
+# Download the data
+# TODO: Add the link to the data once it is available
+
+# pre-process the data
+cd 1.pre_process_data/
+source run_preprocessing.sh
+cd ../
```

Comment:
Seeing the number of `cd` commands made me concerned that this might be challenging to maintain (there are also `cd`'s in the scripts). Would it make sense to remove the nested `cd`'s in the script files sourced here to make directory control "single-sourced"? Generally, where you can, I recommend running all scripts as though they were run from the same location (instead of from within the nested directories), referencing full paths from the context of the project throughout. I bring this up out of care for the development overhead which multiple directory location / relocations can cause.

Reply:
Also references an issue here. This will take refactoring for each notebook